portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -37,6 +39,13 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    UserService userService = UserServiceFactory.getUserService();
+
+    if (!userService.isUserLoggedIn()) {
```

Comment:
Add check for post too? It's strange that posting is allowed but reading is not

Reply:
Done